pkg/apiserver/resthandler.go

```diff
@@ -82,7 +82,13 @@ func GetResource(r RESTGetter, ctxFn ContextFunc, namer ScopeNamer, codec runtim
 }
 
 func parseSelectorQueryParams(query url.Values, version, apiResource string) (label, field labels.Selector, err error) {
-	label, err = labels.Parse(query.Get("labels"))
+	var labelString string
+	if version == "v1beta1" || version == "v1beta2" {
+		labelString = query.Get("labels")
+	} else {
+		labelString = query.Get("label-selector")
```

Comment:
This code isn't intended to have a long lifespan.  Instead it is intended to patch the existing behavior until we can turn down v1beta1/v1beta2.

Building in a generic architecture for changing these flags seems overkill for this PR.  (I'm happy to centralize the function in a single place, though)